test-apps/display-test-app/package.json

```diff
@@ -67,26 +67,32 @@
     "@itwin/imodels-client-authoring": "^3.0.0",
     "@itwin/imodels-client-management": "^3.0.0",
     "@itwin/map-layers-formats": "workspace:*",
+    "@itwin/object-storage-core": "^2.0.0",
```

Comment:
why did we need to add this as a dep? wasnt this getting auto-installed previously?

Reply:
Looking at previous commit history from a month ago, the dep was placed in a different order, that's why in the changes here it looks like its a new dep, but it actually is an old one. I had to bump the version up from what it was a month ago though, rush install was raising something about version mismatch between 2.0.0 and 1.5.0.
![image](https://github.com/iTwin/itwinjs-core/assets/50554904/e3c97690-187c-4048-890f-96c158d47b2c)